gateway/api/connections/connection_credentials.go

```diff
@@ -194,6 +194,15 @@ func buildConnectionCredentialsResponse(
 				"AWS_ACCESS_KEY_ID=%q AWS_SECRET_ACCESS_KEY=%q aws ssm start-session --target {TARGET_INSTANCE} --endpoint-url %q",
 				accessKeyId, accessSecret, endpoint),
 		}
+	case proto.ConnectionTypeHttpProxy:
+		curlCommand := fmt.Sprintf("curl -H 'Authorization: %s' http://%s:%s/", secretKey, serverHost, serverPort)
```

Comment:
Makes sense to leave hardcoded `http`?

Reply:
I will fix this. I think it does not make sense, but this depends how the server proxy started in the main lets say it starts with tls so it need to be https